src/practice/ts/Components/Store/Add.ts

```diff
@@ -0,0 +1,45 @@
+import Storage from "./storage";
+
+class Add {
+  private formElem: HTMLFormElement;
+  private inputElem: HTMLInputElement;
+  private storageContainer: HTMLUListElement;
+  private storage: Storage;
+  private id: number;
+
+  constructor(
+    formElem: HTMLFormElement,
+    inputElem: HTMLInputElement,
+    storageContainer: HTMLUListElement,
+    storage: Storage
+  ) {
+    this.formElem = formElem;
+    this.inputElem = inputElem;
+    this.storageContainer = storageContainer;
+    this.storage = storage;
+    this.id = 0;
+  }
+
+  addUrlInHistory() {
```

Comment:
이 메서드에서 하는 일은 이벤트 추가니까 어울리는 메서드명으로 수정하면 좋을 것 같습니다.
```suggestion
  attachEvent() {
```